packages/lexical-react/src/LexicalNestedComposer.tsx

```diff
@@ -20,10 +20,20 @@ import {
   LexicalNode,
   LexicalNodeReplacement,
 } from 'lexical';
+import {KlassConstructor, Transform} from 'packages/lexical/src/LexicalEditor';
```

Comment:
We can't use relative paths, these will not work well when we import these packages outside the playground. You want to use the name of the package instead

```suggestion
import type {KlassConstructor, Transform} from 'lexical';
```

Reply:
thankyou good feedback :)